.github/workflows/testing.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
           tarantool-version: ${{ matrix.tarantool-version }}
 
       - name: Install build requirements
-        run: sudo apt-get -y install libsasl2-dev
+        run: sudo apt-get -y install libsasl2-dev libevent-dev
```

Comment:
Did I understand correctly that `libevent-dev` is needed for `test-memcached-capable`?

Reply:
Yep.